index.bs

```diff
@@ -367,16 +369,38 @@ QUIC connection.
 
 The [=agent certificate=] must have the following characteristics:
 
-* 256-bit, 384-bit, or 521-bit ECDSA public key
-* Self-signed
-* Supporting the at least one of the following signature algorithms:
-    * secp256r1_sha256
-    * secp384r1_sha384
-    * secp521r1_sha512
-* Valid for signing
+* 256-bit, 384-bit, or 521-bit ECDSA public key.
+* Self-signed.
+* Supporting the at least one of the [=certificate algorithms=] listed below.
+    * The `AlgorithmIdentifiers` are defined in [[!RFC5480]].
```

Comment:
I confess I'm lost when I read this and the table below. RFC5480 does not seem to define the public key and signature numbers that appear in the table.

If it explains how to compute these numbers, I would clarify with:

```suggestion
    * Public key and signature `AlgorithmIdentifiers` are derived from [[!RFC5480]].
```

If the numbers that appear in the table come from elsewhere, it would be good to reference the source.

Or did we just the mint these numbers somehow?

Reply:
No we didn't invent the numbers :-)

In public key certificates the algorithms and parameters are specified using Object IDs that are defined by IETF specs.  RFC 5480 defines the OIDs for ECDSA public keys and RFC 5758 defines them for ECDSA signature schemes.

These OIDs end up being encoded to binary using the ASN.1 encoding scheme and embedded in the certificate (along with the actual public key parameters which vary key-by-key).

My interpretation of the feedback from @sleevi in #280 was that to be crystal clear we should put the actual binary representations of these OIDs in the spec as DER (i.e., hexadecimal).  But this DER form is not in the RFCs, you have to do that conversion yourself ([tool here](https://misc.daniel-marschall.de/asn.1/oid-converter/online.php)) or parse an actual DER-encoded certificate.

I updated this PR to include both the object names, the OIDs, and the DER representations of the OIDs.  Does that make things clearer?

Since we may drop P384 from the supported certificates in PR #297 anyway,  I didn't fill in the row for P384 yet.  We can land #297 then update this PR accordingly.

Reply:
Many thanks for the explanation.

As long the table is clear that it contains the DER representation of the object identifiers defined in the RFCs, as done in your latest update, the object names and the object identifiers may not be needed since they represent the exact same thing. They are useful for laymen like me to figure things out, but I don't think I'm the targeted audience for this part in any case. Up to you!